File: Задание 2 С++/Console App/cmd-calculator/CmdCalculator/CmdCalculator.cpp
#include <iostream>

using namespace std;

int main() {
    double a = 0;
    double b = 0;
    double result = 0;
    char op = 0;

    cout << "=== CMD Calculator ===" << endl;

    cout << "Enter first number: ";
    cin >> a;

    if (cin.fail()) {
        cout << "Invalid first number!" << endl;
        return 1;
    }

    cout << "Enter operator (+ - * /): ";
    cin >> op;

    cout << "Enter second number: ";
    cin >> b;

    if (cin.fail()) {
        cout << "Invalid second number!" << endl;
        return 1;
    }

    switch (op) {
    case '+':
        result = a + b;
        break;

    case '-':
        result = a - b;
        break;

    case '*':
        result = a * b;
        break;

    case '/':
        if (b == 0) {
            cout << "Division by zero!" << endl;
            return 1;
        }

        result = a / b;
        break;

    default:
        cout << "Unknown operator!" << endl;
        return 1;
    }

    cout << "Result: " << result << endl;

    system("pause");

    return 0;
}

File: Задание 4 Python/CmdCalculator.py
def cmd_calculator():
    print("=== CMD Calculator ===")

    try:
        a = float(input("Enter first number: "))
    except ValueError:
        print("Invalid first number!")
        return

    op = input("Enter operator (+ - * /): ")

    try:
        b = float(input("Enter second number: "))
    except ValueError:
        print("Invalid second number!")
        return

    if op == '+':
        result = a + b
    elif op == '-':
        result = a - b
    elif op == '*':
        result = a * b
    elif op == '/':
        if b == 0:
            print("Division by zero!")
            return
        result = a / b
    else:
        print("Unknown operator!")
        return

    print(f"Result: {result}")

if __name__ == "__main__":
    cmd_calculator()